stripe/api_resources/payment_intent.py

```diff
@@ -4446,12 +4441,6 @@ class CreateParamsPaymentMethodOptionsCardPresent(TypedDict):
             """
             Request ability to capture this payment beyond the standard [authorization validity window](https://stripe.com/docs/terminal/features/extended-authorizations#authorization-validity)
             """
-            request_incremental_authorization: NotRequired[
-                "Literal['if_available', 'never']|None"
-            ]
-            """
-            Request ability to [increment](https://stripe.com/docs/payments/incremental-authorization) for this PaymentIntent.
-            """
```

Comment:
I'm confused - the diff says this is removed but when I check out the branch I see it's still there with the correct "This field was released by mistake and will be removed in the next major version" docstring.

Reply:
Ah, it was already generated by https://github.com/stripe/stripe-python/pull/1102